uint64 timestamp			# time since system start (microseconds)

float32 indicated_airspeed_m_s		# indicated airspeed in m/s

float32 true_airspeed_m_s		# true filtered airspeed in m/s

float32 air_temperature_celsius		# air temperature in degrees celsius, -1000 if unknown

float32 confidence			# confidence value from 0 to 1 for this sensor

float32 aoa      # Angle of attack (radians)

float32 aos      # Angle of sideslip (radians)

float32 airspeed_body_x
float32 airspeed_body_y
float32 airspeed_body_z